src/renderer/components/uielements/assets/assetInfo/AssetInfo.style.tsx

```diff
@@ -32,27 +43,58 @@ export const CoinSubtitle = styled.p`
   text-transform: uppercase;
 `
 
-export const CoinPrice = styled.p`
+export const InfoContainer = styled('div')`
```

Comment:
[curious] Any reason to use `styled('div')` and not `styled.div`?

Reply:
yep. you are right. no reason for that